### DISCUSSION TITLE
Server error after updating with git to stop 3.0.1

### DISCUSSION THREAD

**Discussion by FabbyoMalta** [netbox-community/netbox#7158] [Q&A] [answered]
**Has anyone faced the same problem and already have the solution?**

![2021-09-02_11h57_01](https://user-images.githubusercontent.com/68062842/131872342-89d4166a-0a06-4320-bf32-6d5013cb7acb.png)

**OBS : While running the upgrade.sh script I noticed the following error returned : "Skipping local dependencies (local_requirements.txt not found)"
In fact there is no such file but there is base_requerements.txt and requirements.txt why does this happen?**

![2021-09-02_12h30_26](https://user-images.githubusercontent.com/68062842/131873139-8b7dd205-6cc5-401f-9e19-76c5cf3cdc01.png)

**Comment by baldgeek** [comment]
You don't need one if you don't have any optional requirements.

https://netbox.readthedocs.io/en/stable/installation/3-netbox/#optional-requirements 
<https://netbox.readthedocs.io/en/stable/installation/3-netbox/#optional-requirements>

**Reply by FabbyoMalta** [reply]
Got it, in fact I haven't installed any plugin yet, thanks a lot! But what about the first mistake I'm having on the front?

**Comment by jeremystretch** [comment, marked as answer]
Looks like you're also running into #7149
